core/src/main/java/org/pentaho/di/core/plugins/Plugin.java

```diff
@@ -62,6 +62,7 @@
   private String documentationUrl;
   private String casesUrl;
   private String forumUrl;
+  private String suggestedStep;
```

Comment:
It feels strange having this meta specific to steps generically added to Plugin. I wonder if there is a way to move this to be more specific to steps.